tests.mk

```diff
@@ -92,7 +92,7 @@ test_release:
 .PHONY: test_release
 
 test100:
-	@for i in {1..100}; do make test; done
+	@for i in {1..100}; do make test; go clean -testcache; done
```

Comment:
Why do we need this?

Reply:
Have you run this test? @shumkov ? It does not work without it.

Reply:
@alexanderbez did you guys just forgot to add it in Tendermint?